reputation.php

```diff
@@ -936,21 +970,16 @@
 		$last_updated = $lang->sprintf($lang->last_updated, $last_updated_date);
 
 		// Is this rating specific to a post?
-		$postrep_given = '';
+		$postrep_given = $lang->sprintf($lang->postrep_given_nolink, $user['username']);
```

Comment:
This + lack of `isset()` introduce even more noticeable bugs than the fixed issue - every non-post reputation has now incorrect `Rating given for xxx's post` note and the post reputations which should be hidden lead to blank posts.